src/NServiceBus.RavenDB.AcceptanceTests/Config.cs

```diff
@@ -29,4 +39,14 @@ public void Cleanup()
 
         Console.WriteLine("Deleted '{0}' database", documentStore.DefaultDatabase);
     }
+
+    public static DocumentStore GetDefaultDocumentStore(ReadOnlySettings settings)
```

Comment:
In the acceptance tests, not in library code, we are supplying an ACTUAL DocumentStore, so within the acceptance tests, I see no reason to abstract it behind the interface only to have to cast it back in order to analyze the TransactionRecoveryStorage / ResourceManagerId that are not available on the interface.